_sass/_type.scss

```diff
@@ -4,32 +4,32 @@
     text-wrap: pretty;
   }
 
-  a,
-  a:link,
-  a:visited {
+  :where(a, a:link, a:visited) {
+    --_bg-width: var(--link-bg-width, 0);
+    --_bg-color: var(--link-bg-color, #{$orange});
+
     color: inherit;
     text-decoration: none;
     position: relative;
     font-weight: 700;
-    display: inline-block;
-
-    &::before {
-      content: '';
-      background-color: rgba(251, 147, 6, 0.75);
-      position: absolute;
-      left: 0;
-      bottom: 3px;
-      width: 100%;
-      height: 5px;
-      z-index: -1;
-      transition: all .3s ease-in-out;
+
+    padding-inline: 0.25rem;
+
+    transition: background-size 200ms;
```

Comment:
Is it possible for the transition to expand the height as opposed to the width of the link? (It also feels a little fast, 300ms feels about right)

Reply:
300ms seemed a bit to slow when switched to animate vertically. Went with 250ms.

Reply:
I like it!